pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.isort]
 skip_glob = ["*/deps/*", "*/build/*", "*/obselete/*"]
-known_third_party = ["PIL", "attr", "conf", "cv2", "demo_runner", "git", "hypothesis", "imageio", "magnum", "matplotlib", "numba", "numpy", "pytest", "quaternion", "scipy", "settings", "setuptools", "torch", "torchvision", "tqdm"]
+known_third_party = ["PIL", "attr", "conf", "cv2", "demo_runner", "examples", "git", "hypothesis", "imageio", "magnum", "matplotlib", "numba", "numpy", "pytest", "quaternion", "scipy", "settings", "setuptools", "torch", "torchvision", "tqdm"]
```

Comment:
The functionality that is being imported from `examples` should be moved into habitat_sim.

Reply:
I am not sure why "examples" got added to this list. Is this why isort is failing?

Reply:
If you have pre-commit hooks setup properly, it should just work and update isort automatically. The fact that you were able to upload while failing the linter shows that you didn't do pre-commit install in git root.

Reply:
@dexter1691 Remove that change and it should pass the linter.